dist/stat_data.py

```diff
@@ -210,9 +214,19 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_pages_inuse',
         'pages currently held in the cache', 'no_clear,no_scale'),
     CacheStat('cache_read', 'pages read into cache'),
+    CacheStat('cache_read_lookaside',
+        'pages read into cache requiring lookaside entries'),
```

Comment:
Maybe change this into `lookaside_read` operations and include `lookaside_read_bytes`, grouped along with the inserts.